.drone.yml

```
@@ -15,9 +15,13 @@ pipeline:
       matrix:
         TESTS: jsunit
   vue-build-settings:
-    image: node
+    image: nextcloudci/node:node-1
```

```suggestion
    image: nextcloudci/node:node-4
```

This is the name 🙈 https://hub.docker.com/r/nextcloudci/node/tags